src/box/txn.h

```diff
@@ -893,6 +893,38 @@ txn_is_nop(const struct txn *txn)
 	return txn->n_new_rows + txn->n_applier_rows == 0;
 }
 
+static inline bool
```

Comment:
Nit: the function is relatively big, I would say it doesn't have to be `inline`. Only increases the compilation time this way. Same for `txn_is_fully_remote()`.